src/Microsoft.ML.AutoML/Tuner/EciCfoTuner.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.ML.SearchSpace;
+
+namespace Microsoft.ML.AutoML
+{
+    /// <summary>
+    /// propose hyper parameter using ECI index.
+    /// ECI index is a way to measure the importance of a trainer. A higher ECI means a trainer
+    /// is more likely to be picked.
+    /// </summary>
+    public class EciCfoTuner : ITuner
```

Comment:
>Cfo [](http://example.com/codeflow?start=20&length=3)

what is Cfo stands for?

Reply:
CostFrugal tuner

Reply:
### Resolved

Reply:
> CostFrugal tuner, I'll add that in comment

Thanks, where does the `o` comes from?

Reply:
That `o` means `optimizor` which comes from its original paper, but in ml.net we rename it to `CostFrugalTuner` so I update that class to 
`EciCostFrugalTuner` to match the naming pattern as well.